BaseFilter/sections/specific.txt

```diff
@@ -15210,6 +15210,9 @@ movie-censorship.com##td[width^="160"][align="center"] > div[style*="border: 1px
 tubesex.me/eureka
 enworld.org##ul > li[class="postbitlegacy postbitim postcontainer"]
 pornhd.com##ul.thumbs + div.listing-zone.mobile-zone
+correiobraziliense.com.br###cb-publicidade-rasgado-1
+correiobraziliense.com.br##div[class^="pub-ret"]
+correiobraziliense.com.br##div[class^="pub-hal"]
```

Comment:
If I'm not wrong, elements hidden by these two rules:
```adblock
correiobraziliense.com.br##div[class^="pub-hal"]
correiobraziliense.com.br###cb-publicidade-rasgado-1
```
are already hidden by this rule:
```adblock
correiobraziliense.com.br##div[id^="cb-publicidade-"]
```
from `AdGuard Spanish/Portuguese filter`.
Am I wrong?

<hr>

Regarding:
```adblock
correiobraziliense.com.br##div[class^="pub-ret"]
```
maybe it could be simplified to:
```adblock
correiobraziliense.com.br##.pub-ret
```

Reply:
And one more thing, it should be moved to `AdGuard Spanish/Portuguese filter` - https://github.com/AdguardTeam/AdguardFilters/blob/master/SpanishFilter/sections/specific.txt